crates/viewer/re_viewport_blueprint/src/view_properties.rs

```diff
@@ -103,8 +103,6 @@ impl ViewProperty {
             .ok_or(ComponentFallbackError::UnexpectedEmptyFallback.into())
     }
 
-    // TODO(#6889): It's a bit sad that we can't derive the type of `C` from the `ComponentDescriptor`, which means
```

Comment:
guess we all just grew beyond this sadness and started embracing our type erased reality 😄